src/pytezos/rpc/node.py

```diff
@@ -74,6 +95,7 @@ def __repr__(self):
         ]
         return '\n'.join(res)
 
+    @retry
     def request(self, method, path, **kwargs) -> requests.Response:
```

Comment:
I'd exclude `inject` endpoints (or for simplicity all non-GET requests) just in case.